### DISCUSSION TITLE
All-Time Tokens Statistic

### DISCUSSION THREAD

**Discussion by ZenofyMedia** [shirokumacode/overwatch-omnic-rewards#27] [Ideas]
Pretty much as the title suggests, it'd be cool to see the amount of Tokens you've earned through Omnic Rewards.

**Comment by shirokumacode** [comment]
Agreed.

Just added this to the app, commit https://github.com/shirokumacode/overwatch-omnic-rewards/commit/9e66cd1cfc97e780c9cfee0bd4b688e648dd499e. Will be out on the next release. Don't really know when it will be given that such a small added thing isn't enough for a new release.
